src/ert/shared/cli/main.py

```diff
@@ -97,17 +98,18 @@ def run_cli(args):
         model, ee_con_info=evaluator_server_config.get_connection_info()
     )
 
-    out = open(os.devnull, "w") if args.disable_monitoring else sys.stderr
-    monitor = Monitor(out=out, color_always=args.color_always)
-
-    try:
-        monitor.monitor(tracker)
-    except (SystemExit, KeyboardInterrupt):
-        print("\nKilling simulations...")
-        tracker.request_termination()
-
-    if args.disable_monitoring:
-        out.close()
+    with contextlib.ExitStack() as exit_stack:
```

Comment:
I don't understand if using `contextlib` here would change the behaviour. How did you get to this new import here?

Reply:
This is based on the docs in:
 https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack

We need the exit stack here just because the file is only opened `if args.disable_monitoring` and then using a standard `with open...` is not doable.